packages/storybook/src/stories/use-toast.mdx

```diff
@@ -0,0 +1,69 @@
+import { Canvas, Meta, Story } from '@storybook/addon-docs';
+import * as ToastStories from './use-toast.stories.svelte';
+
+<Meta title='APIs/useToast' />
+
+# useToast
+
+Components and an API for displaying user toast messages.
+
+## Setup
+
+To use toasts, provide the toast context somewhere high your component tree along with the `ToastContainer` component.
+
+The root layout component is probably the best place to set up the context.
+
+```svelte
+<!-- +layout.svelte -->
+<script lang="ts">
+import { ToastContainer, provideToast } from '@viamrobotics/prime-core';
+
+import PageRoot from '$lib/layout/page-root.svelte';
+import PageContent from '$lib/layout/page-content.svelte';
+
+provideToast();
+</script>
+
+<PageRoot>
+  <PageContent>
+    <slot />
+  </PageContent>
+  <ToastContainer />
+</PageRoot>
+```
+
+## Usage
+
+To display a toast, use the `useToast` function:
+
+```ts
+import { useToast } from '@viamrobotics/prime-core';
+
+const doSomethingCrazy = () => {
+  const toast = useNotify();
+
+  notify.neutral('Hello, Brooklyn!');
+};
+```
+
+There are three different types of toast, each with a different styling.
+
+```ts
+import { useToast } from '@viamrobotics/prime-core';
+
+const doSomethingCrazy = () => {
+  const toast = useToast();
+
+  toast.neutral('Plain slice please!');
+  toast.upload('Uploading file!');
+  toast.success('Fuhgeddaboudit!');
```

Comment:
I know this comes from the `notify` usage, but I think it's a great time to reimagine this. For me, the level seems better as an options argument rather than having separate methods for each level

Reply:
So update the API to something like this?

```
  toast('Plain slice please!');
  toast('Uploading file!', ToastVariant.Upload);
  toast('Fuhgeddaboudit!', ToastVariant.Success);
```

Reply:
Yeah, though I'd use an options object, as Ethan suggested in his review. So something like:

```ts
toast({ message: 'Hello world', variant: ToastVariant.SUCCESS })
```

From there, if we wanted, we could layer stuff in like "default to `INFO`" and "allow a plain string to toast an info message"